### ISSUE TITLE
Make a task list for novices, do prep work to make them actionable

### ISSUE THREAD

**Issue by GrumpyOldTroll** [cvuuf/church#89]
We're going to have a website-editing party, where 10 or so people from a chalice circle get together and spend a day doing some website cleanup.  This was selected by one of the chalice circles as their service project for the year.  Likely date is perhaps sometime in April, but not yet selected.

It's useful to prepare some stuff they can usefully engage with.  A couple of initial ideas:
- task for party: have them review the pages on the site for grammar and content.  prep work: compile a list of urls of all the pages, to divide among the participants.
- task for party: have them search thru photos to update the pages, matched against the guidelines from the uua's suggested image list for the theme:
  https://www.uua.org/communications/websites/wordpress-theme/documentation/content/suggested-images
  We have a few photo archives, I'll get a list together and send an email.  People can also bring some photos they think they can use.

There are likely some other ideas that might be better, add them in comments if you think of any.  Goal 1 is to do something useful to make the website nicer, and goal 2 is to recruit new people who are comfortable making website edits, and see the value it adds for the fellowship to give it some attention.

**Comment by chuckfernald** [comment]
I will compile a list of URLs. 
Do we need membership area URLs or just public ones?

**Comment by shaneniebergall** [comment]
I've been thinking about this is and having a group of people all editing the website that have no prior experience scares me a little. I like your idea of having them find photos, or point out things that are broken or need updating, but I'd vote that they compile a list and then we end up making the changes.

They could also come up with a 'wish list' for the website (and they vote and pick their top X) for things that we could implement in the near future.

**Comment by chuckfernald** [comment]
I agree with Shane on this.

**Comment by GrumpyOldTroll** [comment]
I agree we should use caution here, and that I don't want people breaking things.

I may have explained this a little too loosely; I think one of the reasons the chalice circle went for it was that several people do have some website-editing experience, and in some cases some wordpress experience, even though they have not yet edited our website.  (Those with some experience included Kendra, Raul, Brandi, and I think Mike, IIRC, as well as David, and of course Kelly and me.)

There were also a few folks who expressed some reluctance and worried they'd mess something up.  These folks said they could take on more hosting tasks for the party, like the food, cleaning, etc., and perhaps not do much or any actual website editing, which sounded like a good kind of nervousness from people who were really inexperienced or uncomfortable with technology.

So the reason I think it's probably ok for this chalice circle is that it skews a little toward tech-savvy and editing experience, and those who weren't that way were eager to step backwards or find another thing to do.

I'm expecting the chaos will be reasonably manageable, especially if we have them documenting what they do.  Like maybe first we teach them opening github issues, including the link to the page they're touching, and screenshotting the issue, and then we walk people through making the changes.

Also, I was going to give them probably editor roles instead of admin, so they can't accidentally muck up most of the plugins or anything.  Maybe some other countermeasures would also be appropriate?  I meant to probably to have one of us over the shoulder on most folks doing their first few edits (if there's only 6 such in the circle, and the first few edits are small, this seems doable I think?)

So I'd say I'm only a little bit scared of what can go wrong here, and I think the upside potential is pretty good if we can find someone who enjoys it.  

That said, I will listen to appeals for caution, and happily accept suggestions that reduce risk, and I think it's a good idea to focus first on things that can't badly break anything, like photo-sorting and maybe github issue-opening, and moving on only where people are demonstrating a reasonable level of comfort and interest.  But I think "plan not to have any of them do any edits" might be a bit too pessimistic if we have them for much of a day.

**Comment by shaneniebergall** [comment]
Another good page to update at party: https://www.chaliceuu.org/about-us/our-stories/
